conda-recipe/run_test.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 # Suppress error b/c it could fail on Ubuntu 18.04
 source ${ONEAPI_ROOT}/compiler/latest/env/vars.sh || true
+source ${ONEAPI_ROOT}/tbb/latest/env/vars.sh || true
```

Comment:
@PokhodenkoSA 
You hiding the error in compiler activation script (about FPGA), I disagree with it but ok... I think that explicit fix of scripts or renaming of oclfpga directory is better.
But why you need `|| true` in tbb activation line?

Reply:
Generally to avoid failing the script if sourcing the TBB's `vars.sh` returns with non-zero code.

Reply:
It does not help anyway. Sourcing TBB does not make CPU device work.

Reply:
It happens because someone changed "latest" compiler with beta10 version.
TBB has beta09 version, so it is incompatible.
source another version of compiler and it will work.

Reply:
I mean, device itself will be available in clinfo for example, but error -2 still occurs...